widgetssdk/src/main/java/com/glia/widgets/call/CallController.kt

```diff
@@ -180,8 +180,8 @@ internal class CallController(
         initControllerCallbacks()
         initMessagesNotSeenCallback()
         tryToQueueForEngagement(queueId)
-        inactivityTimeCounter.addRawValueListener(inactivityTimerStatusListener)
-        connectingTimerCounter.addRawValueListener(connectingTimerStatusListener)
+        inactivityTimeCounter.addRawValueListener(inactivityTimerStatusListener!!) // Listener is created few lines before
```

Comment:
Honestly, I don't like to use `!!`.
The only place I would do it is in tests.

Reply:
And where is it a few lines before?
We could use the value here without the unsafe operator if a method were return the callback object.

Reply:
I will look into how to improve this.

Reply:
Fixed in 4a01134